docs/ARDUPILOT.md

```diff
@@ -0,0 +1,55 @@
+# mLRS Documentation: ArduPilot Setup #
+
+([back to main page](../README.md))
+
+([back to CRSF page](CRSF.md))
+
+Further parameters can be configured to optimize peformance for ArduPilot and the RF mode (19Hz, 31Hz or 50Hz) selected.
+
+## mLRS Rx Module Setup
```

Comment:
this piece IMHO should be in CRSF.
here I would expect more explanations, like e.g. ardu_1 and brad

Reply:
- Added all serial parameters to CRSF page.
- Added spot on AP page for explanation of ardu_1, brad options